evalai/get_token.py

```diff
@@ -0,0 +1,24 @@
+import click
+import os
+
+from click import echo, style
+
+from evalai.utils.config import AUTH_TOKEN_PATH
+
+
+@click.group(invoke_without_command=True)
+def get_token():
+    """
+    Get the EvalAI token.
+    """
+    if not os.path.exists(AUTH_TOKEN_PATH):
+        echo(style("\nThe authentication token json file doesn't exist at the required path. "
+                   "Please download the file from the Profile section of the EvalAI webapp and "
+                   "place it at ~/.evalai/token.json or use evalai -t <token> to add it.\n\n", bold=True))
+    else:
+        with open(AUTH_TOKEN_PATH, 'r') as fr:
+            try:
+                data = fr.read()
+                echo("Current token is {}".format(data))
```

Comment:
It should display `The current auth token is {}.format(data)` where data is token only.

Reply:
@nileshprasad137 Please change the text too!